content/introduction/how-ipfs-works.md

```diff
@@ -0,0 +1,88 @@
+
+It's the same on the internet and on your computer. Right now, content is found by location, such as…
+
+- `https://en.wikipedia.org/wiki/Aardvark`
+- `/Users/Alice/Documents/term_paper.doc`
+- `C:\Users\Joe\My Documents\project_sprint_presentation.ppt`
+
+By contast, every piece of content that uses the IPFS protocol has a [*content identifier*]({{<relref "guides/concepts/cid.md">}}) that is its *hash*. The hash is unique to the content that it came from, even though it’s short (well, short compared to all the bits and bytes that make up the original content). _If hashes are new to you, check out [the concept guide on hashes]({{<relref "guides/concepts/hashes.md">}}) for a good introduction._
+
+Content addressing through hashes has become a widely-used means of connecting data in distributed systems, from the blockchains that run your favorite cryptocurrencies, to the commits that back your code, to the web’s content at large. However, the underlying data structures in these systems are not necessarily interoperable.
+
+This is where the [IPLD project](https://ipld.io/) comes in. **Hashes identify content, and IPLD links that content together** using any number of arrangements. It's a bit like your file system at home. You might organize all your bills by year; another person might organize them by type. These are both valid ways of organizing content, but you have to know which one you're using, otherwise you can't find anything or add anything to your folders.
+
+IPLD is "software" that provides both rules for how to organize content-addressed data, and translations across different organization methods. Since different distributed systems structure their data in different ways, IPLD provides a platform for combining pluggable modules (parsers for each possible type of IPLD node) to resolve a path, selector, or query across many linked nodes (of varying formats, even!). IPLD really refers to any content addressable data with links to other content addressable data.
+
+In the case of a peer to peer network using the IPFS protocol, your computer uses "IPFS-flavored IPLD" to get from raw content to an IPFS address. IPFS has its own preferences and conventions about how data should be broken up into a DAG (more on DAGs below!); IPLD links content on the IPFS network together using those conventions.
+
+**Everything else in the IPFS ecosystem builds on top of this core concept; linked, addressable content is the fundamental connecting element that makes the rest work.**
+
+### 2 \\ DAGs
```

Comment:
It felt like a chicken-or-egg problem because they're so interconnected -- I went back and forth several times. In the end, this is where I landed, but there might be a better way that will reveal itself as we keep editing (and others review).

Reply:
ack